BATCH_MANUAL.md

```diff
@@ -0,0 +1,93 @@
+# LDAR-Sim Batch Running Manual
+
+The batch runner offers an alternative method for executing LDAR-Sim. With this feature, users can run the standard simulation in multiple batches, consolidating the outputs of these batches into a single averaged value whenever applicable
+
+## How to use
+
+To utilize the batch script, follow these steps to configure LDAR-Sim with all parameters in a designated directory. Then, employ the following command line structure:
+```ldar_sim_batchrun.py --n_rep {number_of_repetitions} -P {directory_path}```
+
+Example command to run 100 batch simulations contained in the folder simulations would be:
+```ldar_sim_batchrun.py --n_rep 100 -P simulations```
+
+## Outputs
+
+### Regular Outputs
+
+The following files are maintained for each of programs of the first batch of simulations
+```timeseries_output_{simulation}_{Program name}.csv```
+```leaks_output_{simulation}_{Program name}.csv```
+
+The following file will be kept for the very last batch of simulations that are run.
+```sites_output_{simulation}_{Program name}.csv```
+
+### Sites
+
+`sites_output_{program}_concat.csv`
+
```

Comment:
Additional content: Can you include the Additional Statistics column as well and describe what might be in there